content/_index.md

```diff
@@ -30,14 +30,16 @@ We are excited to announce that BSidesTLV 2020 will take place in hybrid format!
       icon="cfp" >}}
 
     {{< button-link
-      url="//tickets.bsidestlv.com/bsidestlv/2020"
+      url="//tickets.bsidestlv.com/bsidestlv/2020/5/"
```

Comment:
the register button on the main page should lead to the registration page which has the sponsor logos

Reply:
not related to this issue